src/components/filterType/FilterType.js

```diff
@@ -3,20 +3,16 @@ import React from 'react';
 // css
 import './FilterType.css'
 
-function FilterType({ heading, inputs, setFilters }) {
+function FilterType({ heading, inputs, setFilters, isChecked, setIsChecked }) {
     const updateCategory = (filters, input) => {
+        !filters.category.includes(input) ? setIsChecked([...isChecked, input]) : setIsChecked(isChecked.filter(data => data !== input))
         return {
             ...filters,
             category: filters.category.includes(input) ? filters.category.filter(data => data !== input) : [...filters.category, input]
         }
     }
-    const updateSort = (filters, input) => {
-        return {
-            ...filters,
-            sort: filters.sort.includes(input) ? filters.sort.filter(data => data !== input) : [input]
-        }
-    }
     const updateRating = (filters, input) => {
+        !filters.rating.includes(input) ? setIsChecked([...isChecked, input]) : setIsChecked(isChecked.filter(data => data !== input))
```

Comment:
Suggestion - for filters and rating if you are using the same function, then you can call this function seperately in both the updateRating and updateFilter.

Reply:
can be done, thanks for suggestion